viewing_party/party.py

```diff
@@ -0,0 +1,151 @@
+
+#Wave 1
+
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating:
+        movie_dict = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating}
+        return movie_dict
+    else:
+        return None
+
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+    for movie in watchlist:
+        if movie["title"] == title:
+            title = movie
+    if title in user_data["watchlist"]:
+        user_data["watchlist"].remove(title)
+        user_data["watched"].append(title)
+            
+    return user_data
+
+# Wave 2
+
+def get_watched_avg_rating(user_data):
+    watched = user_data["watched"]
+    sum=0
+    count=0
+    if watched == []:
+        print(watched)
+        return 0.0
+    else:
+        for movie in watched:
+            sum+=movie["rating"]
+            count+=1
+        avg=sum/count
+        return avg
+
+
+def get_most_watched_genre(user_data):
+    watched = user_data["watched"]
+    if watched == []:
+        return None
+    else:
```

Comment:
As above, consider moving this code outside the else block to reduce indentation and enhance readability